pyproject.toml

```diff
@@ -19,9 +19,10 @@ classifiers = [
     'Programming Language :: Python :: 3.11',
     'Programming Language :: Python :: 3.12',
     'Programming Language :: Python :: 3.13',
+    'Programming Language :: Python :: 3.14',
     'Typing :: Typed'
 ]
-dependencies = []
+dependencies = ['tzdata']
```

Comment:
One of the goals of this project was to not have any dependencies outside the standard library. Why is this one needed now?